integration_test/third_party_apps_data/applications/mysql/expected_metrics.yaml

```diff
@@ -9,13 +9,29 @@
   kind: GAUGE
   monitored_resource: gce_instance
   labels:
-    kind: data|dirty|flushed|free|misc|total
+    kind: data|free|misc
+- type: workload.googleapis.com/mysql.buffer_pool_data_pages
+  value_type: INT64
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    kind: dirty|flushed
+- type: workload.googleapis.com/mysql.buffer_pool_page_flushes
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels: []
```

Comment:
Empty `labels` should be `{}` (there's another one below for `buffer_pool_limit`)